pom.xml

```diff
@@ -23,7 +23,7 @@
   </parent>
 
   <artifactId>vertx-core</artifactId>
-  <version>3.5.0-SNAPSHOT</version>
+  <version>3.5.0-SNAPSHOT-CMP</version>
```

Comment:
No, I was using this marker for my local branch to differentiate with the actual current version. Will back it out.